### ISSUE TITLE
Mclennan: Add write toggle to enable setting fields non-permanently without setting macros

### ISSUE THREAD

**Issue by rerpha** [ISISComputingGroup/IBEX#8230]
Currently the Mclennan uses a set of macros (JVEL1 for Jog VELocity, axis 1, etc) to configure things like limits and velocities. 
I would like to add a PV that toggles the access security that stops a user from setting these fields on the motor record directly (ie. setting MTRxx01.JVEL instead of setting JVEL1) 

### Acceptance criteria
- [ ] Toggle PV is added to mclennan IOC which disables the access security (https://github.com/ISISComputingGroup/EPICS-ioc/blob/master/MCLEN/MCLEN-IOC-01App/Db/mclen_motor_extra.db#L13) and enables it. This should default to enabling the security so by default a user cannot change eg. JVEL on the fly from the ToM
- [ ] GUI is changed so that the toggle is shown in manager mode on the ToM basic and more details page in the same place that the beckhoff errors/galil errors/extra controls are shown

**Comment by FreddieAkeroyd** [comment]
@rerpha do they need access security toggle to enable setting everything, or is there a particular subset  that would serve their case. I believe one of the reasons they were protected was to avoid confusion during setup and then things not getting remembered, if the ioc always starts with access disabled (i.e. toggle choice is not autosaved or in a config macro) then that may be sufficient

**Comment by rerpha** [comment]
Just a subset - limits and velocities being important: 

```
Yes I would like to be able to set jog velocity and motor limits on the fly. The example code below used to work but now does not work anymore on LET after the latest IBEX update:

motor_pv="MOT:MTR0201"
 g.set_pv(f"{motor_pv}.JVEL", 0.01, is_local=True, wait=True)
 g.set_pv(f"{motor_pv}.HLM", 180, is_local=True, wait=True)
 g.set_pv(f"{motor_pv}.LLM", -180, is_local=True, wait=True)

If we can allow those fields to be set again in a script that would be helpful. I agree that things like motor resolution should not be allowed to be changed that way as this should stay fixed for each motor.
```

**Comment by FreddieAkeroyd** [comment]
Using manager mode requires remembering to enable it afterwards and then forgetting may affect a script. Other options are to:
- (1) make accecss security choice a macro add a warning next to settings on a screen tellign user they aer non-persistent 
- (2) leave access security on, but add backdoor PVs for a few settings that can be used as above

**Comment by KathrynBaker** [comment]
This was discussed at SAG, and the approach that is acceptable to our users would be to create a macro that will overwrite the security group for the PVs above, however, this macro shout NOT be made available in config.xml, but is one to set in globals.txt. This can be enacted and patched, so has been created as #8284 

The second action is to make sure that there is something visible on the motor screen to say something along the lines of "Changes made here will not persist, please make changes to the macros for the IOC if the changes should be reapplied next time IBEX is loaded." This message should only be visible when that macro is set to allow the values to change. This has been created as #8285
